compiler/circle-mpqsolver/CMakeLists.txt

```diff
@@ -1,9 +1,18 @@
+nnas_find_package(Jsoncpp)
+if(NOT Jsoncpp_FOUND)
```

Comment:
Lets split this change as this is not directly related with VISQErrorApproximator

Reply:
ok

Reply:
@seanshpark 
This one is needed to build VISQErrorApproximator.

Reply:
you can split `VISQErrorApproximator` to parts.

Reply:
@seanshpark 
May i  suggest the following sequence of PR's:
1. disabled tests, no `json`
2. `json`'s part and enavling tests
?
Or may be there is another more preferable sequence?

Reply:
anyway to meet `introduce VISQErrorApproximator`

Reply:
@seanshpark 
seems like it's fixed.